### PR TITLE
chore: making copy:snippets run on windows

### PR DESCRIPTION
**Pull request by pikax**
instead of using bash converted the script to javascript

### CODE REVIEW

File: build/copy-snippets.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const rimraf = require('rimraf');
+const { resolve } = require('path');
```
Comment (octref) [inline build/copy-snippets.js:2]: Unused import

File: build/copy-snippets.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+const { ncp } = require('ncp');
+const { ensureDirSync } = require('fs-extra');
+
+const PDIR = 'server/dist/modes/vue';
+const DIR = 'server/dist/modes/vue/veturSnippets';
+
+ensureDirSync(PDIR);
```
Comment (octref) [inline build/copy-snippets.js:9]:
```suggestion
// Create parent dir and ensure clean snippet dir
ensureDirSync(PDIR);
```

File: test/lsp/codeAction/basic.test.ts [hunk 3]
```diff
@@ -17,6 +17,8 @@ describe('Should do codeAction', () => {
   });
 
   it('finds codeAction for unused import', async () => {
+    await sleep(2000); // waiting more, on windows is really slow
```
Comment (octref) [inline test/lsp/codeAction/basic.test.ts:20]: Are you sure this is the cause? I think we should not add more random sleep but improve Windows integration test.

Let's not make this change just to pass CI. If it's flakey you can run it multiple times by closing/reopening.

Comment (yoyo930021) [reply test/lsp/codeAction/basic.test.ts:20]: I think #1789 can stable this problem.

Comment (pikax) [reply test/lsp/codeAction/basic.test.ts]: I'm using Windows and I can't reproduce this issues, I agree a better approach is desired

File: test/lsp/codeAction/basic.test.ts [hunk 4]
```diff
@@ -43,9 +43,9 @@ async function testCodeAction(docUri: vscode.Uri, range: vscode.Range, expectedA
     range
   )) as CodeAction[];
 
-  expectedActions.forEach(eAction => {
+  expectedActions.forEach((eAction) => {
```
Comment (octref) [inline test/lsp/codeAction/basic.test.ts:46]:
```suggestion
  expectedActions.forEach(eAction => {
```

Comment (octref) [reply test/lsp/codeAction/basic.test.ts:46]: I removed this change. Sorry. I have just updated prettier config in master so this shouldn't happen again.

**Comment by octref** [comment]
LGTM, thanks.

**Pull request merged**